cfgov/jinja2/v1/_layouts/content-base-sidebar-first.html

```diff
@@ -1,5 +1,9 @@
 {% extends 'content-base.html' %}
 
+{% block skip_nav_target -%}
```

Comment:
I recommend adding comments to explain what this does for future us

Reply:
Good idea! Added in an amended, rebased commit and force-pushed.